M2/Macaulay2/packages/Macaulay2Doc/functions/check-doc.m2

```diff
@@ -5,21 +5,13 @@
 doc ///
 Node
   Key
-     TEST
-    (TEST, String)
-    (TEST, List)
-    [TEST, FileName]
+     symbol TEST
```

Comment:
I haven't tested this, do both `help symbol TEST` and `help "TEST"` work this way? I think other keywords are usually documented as strings.

Reply:
Yes, they both work